website/pages/docs/connect/proxies/envoy.mdx

```diff
@@ -319,6 +319,10 @@ will continue to be supported.
 - `envoy_gateway_no_default_bind` - Prevents binding to the default address
   of the gateway service. This should be used with one of the other options
   to configure the gateway's bind addresses.
+  
+- `envoy_dns_discovery_type` - Determines how Envoy will resolve hostnames. Defaults to `LOGICAL_DNS`.
```

Comment:
Should we clarify that this is only used for external services via a terminating gateway right now?

Reply:
This applies to both mesh gateways and terminating gateways, so I added a note about that.